docs/reference/channel.md

```diff
@@ -508,6 +508,11 @@ that it will cause your pipeline to run forever. Consider using the `take` or `u
 a certain condition is met (e.g. after receiving 10 files, receiving a file named `DONE`).
 :::
 
+:::{note}
+The `channel.watchPath` relies on the Java file watch API which and POSIX notify events.
+As such, `channel.watchPath` may not work with NFS network mounts and will not work for cloud provider URIs (e.g. `s3://.../*.txt`).
```

Comment:
What's the issue with NFS?

Reply:
Per Paulo's comment here: https://github.com/nextflow-io/nextflow/issues/1061#issuecomment-933253132

Perhaps there is something better than "may not work"... it may depend on specific implementations. Unfortunately, I don't have many systems to test specific scenarios.

Reply:
As of that comment theres' a new implementation that's safe with NFS https://github.com/nextflow-io/nextflow/blob/19c40a4a89c6357d3c0ee1c524e503a1baeebe25/modules/nextflow/src/main/groovy/nextflow/file/DirWatcherV2.groovy#L42-L42

Reply:
Nice! It's just a small note about URIs then.